dashboard/app/models/level.rb

```diff
@@ -16,6 +16,7 @@
 #  md5                      :string(255)
 #  published                :boolean          default(FALSE), not null
 #  notes                    :text(65535)
+#  audit_log                :text(65535)
```

Comment:
Since our audit log is ever-increasing and contains changed properties, you just _know_ we're going to hit 65k characters of JSON faster than we expect.  Our `log_changes` hook should probably be smart enough to drop the oldest changes in the log if it's going to exceed this limit, or have some other 'expiration' that will keep level logs from growing indefinitely.